build fail looks real:
```

======================================================================
FAIL: test_all_pillows_handle_errors (pillow_retry.tests.PillowtopRetryAllPillowsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/pillow_retry/tests.py", line 296, in test_all_pillows_handle_errors
    self._test_error_logging_for_pillow(pillow_config)
  File "corehq/ex-submodules/pillow_retry/tests.py", line 304, in _test_error_logging_for_pillow
    self.assertEqual(1, len(errors), pillow_config)
AssertionError: PillowConfig(section='core_ext', name='kafka-ucr-main', class_name='corehq.apps.userreports.pillow.ConfigurableReportKafkaPillow', instance_generator='corehq.apps.userreports.pillow.get_kafka_ucr_pillow')
----------------------------------------------------------------------
```

--- a/corehq/apps/userreports/pillow.py
+++ b/corehq/apps/userreports/pillow.py
@@ -136,6 +136,10 @@ class ConfigurableReportKafkaPillow(ConstructedPillow):
     # we could easily remove the class and push all the stuff in __init__ to
     # get_kafka_ucr_pillow below if we wanted.
 
+    # don't retry errors until we figure out how to distinguish between
+    # doc save errors and data source config errors
+    retry_errors = False
+
     def __init__(self, processor, pillow_name):
         change_feed = KafkaChangeFeed(topics.ALL, group_id=pillow_name)
         checkpoint = PillowCheckpoint(pillow_name)

--- a/corehq/ex-submodules/pillow_retry/tests.py
+++ b/corehq/ex-submodules/pillow_retry/tests.py
@@ -297,6 +297,9 @@ class PillowtopRetryAllPillowsTests(TestCase):
 
     def _test_error_logging_for_pillow(self, pillow_config):
         pillow = _pillow_instance_from_config_with_mock_process_change(pillow_config)
+        if not pillow.retry_errors:
+            return
+
         doc = self._get_random_doc()
         pillow.process_with_error_handling(Change(id=doc['id'], sequence_id='3', document=doc))
 

--- a/corehq/ex-submodules/pillowtop/pillow/interface.py
+++ b/corehq/ex-submodules/pillowtop/pillow/interface.py
@@ -27,6 +27,9 @@ class PillowBase(object):
     """
     __metaclass__ = ABCMeta
 
+    # set to true to disable saving pillow retry errors
+    retry_errors = True
+
     @abstractproperty
     def pillow_id(self):
         """
@@ -171,16 +174,8 @@ class ConstructedPillow(PillowBase):
 
 def handle_pillow_error(pillow, change, exception):
     from pillow_retry.models import PillowError
-    save_error = True
-    try:
-        from corehq.apps.userreports.pillow import ConfigurableReportKafkaPillow
-        if isinstance(pillow, ConfigurableReportKafkaPillow):
-            save_error = False  # this is temporarily not supported!
-    except ImportError:
-        pass
-
     error_id = None
-    if save_error:
+    if pillow.retry_errors:
         error = PillowError.get_or_create(change, pillow)
         error.add_attempt(exception, sys.exc_info()[2])
         error.save()

